app/models/health.rb

```diff
@@ -0,0 +1,88 @@
+class Health
+  # things to check: delayed_job, background crons, database, ECHO, CMR, URS, Opensearch, browse-scaler
+
+  def initialize
+    @ok = true
+  end
+
+  def edsc_status
+    {ok?: @ok}
+  end
+
+  def delayed_job_status
+    job = DelayedJob.last
+    retrieval = Retrieval.last
+    if job.run_at > retrieval.created_at
```

Comment:
I think this needs a grace period of a couple of minutes, or you run the risk of reporting a false positive the moment after a retrieval is queued, before the queue gets checked by the background job. Maybe select the first retrieval that's more than 5 minutes old?